NickvisionTagger.GNOME/Controls/CreatePlaylistDialog.cs

```diff
@@ -92,24 +92,34 @@ private void Validate()
     /// <param name="e">EventArgs</param>
     private async void SelectSaveLocation(Gtk.Button sender, EventArgs e)
     {
+        var playlistExtensions = Enum.GetValues<PlaylistFormat>().Select(x => x.GetDotExtension()).ToList();
         var fileDialog = Gtk.FileDialog.New();
         fileDialog.SetTitle(_("Save Playlist"));
         var filters = Gio.ListStore.New(Gtk.FileFilter.GetGType());
         var filterAll = Gtk.FileFilter.New();
         filterAll.SetName(_("All Files"));
-        foreach (var format in Enum.GetValues<PlaylistFormat>())
+        foreach (var ext in playlistExtensions)
         {
-            var extension = format.GetDotExtension();
-            filterAll.AddPattern($"*{extension}");
-            filterAll.AddPattern($"*{extension.ToUpper()}");
+            var filter = Gtk.FileFilter.New();
+            filter.SetName($"{ext.Replace(".", "").ToUpper()} (*{ext})");
```

Comment:
I suggest `_("{0} Playlist (*{1})", ext.Replace(".", "").ToUpper(), ext`, so it will be `M3U Playlist (*.m3u)`

Reply:
Done in latest commit